MFW-458: Add Rules List example for Magic Firewall

arges opened the pull request.

dcpena commented:
This reads to me as more of a procedure/how to use account-level rules lists, so I'd suggest adding this under the How to section. Content looks good overall though!

arges commented:
> This reads to me as more of a procedure/how to use account-level rules lists, so I'd suggest adding this under the How to section. Content looks good overall though!

@dcpena addressed comments
